dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/_source/dependItemList.vue

```diff
@@ -137,15 +140,21 @@
       },
       _getProcessByProjectCode (code) {
         return new Promise((resolve, reject) => {
-          this.store.dispatch('dag/getProcessByProjectCode', code).then(res => {
-            let definitionList = _.map(_.cloneDeep(res), v => {
-              return {
-                value: v.processDefinition.code,
-                label: v.processDefinition.name
-              }
+          let definitionCacheList = JSON.parse(sessionStorage.getItem(projectCodePrefix + code))
+          if (definitionCacheList) {
```

Comment:
If sessionStorage is used here, and the user adds a workflow after caching, the drop-down list may not display the new workflow?